R/qenv-constructor.R

```diff
@@ -1,24 +1,52 @@
 #' Initialize `qenv` object
 #'
-#' Initialize `qenv` object with `code` and `env`. In order to have `qenv` reproducible
-#' one needs to initialize with `env` which can be reproduced by the `code`. Alternatively, one
-#' can create an empty `qenv` and evaluate the expressions in this object using `eval_code`.
-#' @name new_qenv
+#' Create an empty `qenv` object.
 #'
-#' @param code (`character(1)` or `language`) code to evaluate. Accepts and stores comments also.
-#' @param env (`environment`) Environment being a result of the `code` evaluation.
+#' `qenv()` instantiates a `qenv` with an empty environment.
+#' Any changes must be made by evaluating code in it with `eval_code` or `within`, thereby ensuring reproducibility.
+#'
+#' `new_qenv()` can instantiate a `qenv` object with data in the environment and code registered.
+#' This method is deprecated.
```

Comment:
Let's not give user a room to think for a second that this method could be used for something.
 
```suggestion
#' `new_qenv()` (deprecated) could instantiate a `qenv` object with data in the environment and code registered.
```